src/domain/mongoose_domain_core.erl

```diff
@@ -46,7 +46,11 @@
 start() ->
     Pairs = get_static_pairs(),
     AllowedHostTypes = mongoose_config:get_opt(host_types),
-    start(Pairs, AllowedHostTypes).
+    ChildSpec =
```

Comment:
Is testing version of start/2 used anywhere?
Please do not duplicate ChildSpec definition, start/0 should call start/2 with default parameters.